sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -508,8 +508,6 @@ public static int getMaxItemCountForVectorSearch() {
     }
 
     public static boolean getAzureCosmosNonStreamingOrderByDisabled() {
-        logger.info("AZURE_COSMOS_DISABLE_NON_STREAMING_ORDER_BY property is: {}", System.getProperty(AZURE_COSMOS_DISABLE_NON_STREAMING_ORDER_BY));
```

Comment:
This change should have created a merge conflict - the logging part has been addressed already. Please pull Main and check whatever the auto-conflict resolution did to make sure this works out.

Reply:
main cannot be merged in directly, since this is Spring Boot 3 branch. But I agree, this change should eventually get here.

Reply:
Alternative is ot exactly port the change from Main to avoid the merge conflict later.

Reply:
Makes sense, we should do that.